client/html/blocks/menu/menu.html

```diff
@@ -1,4 +1,4 @@
-<nav class="menu page__menu" ng-class="{menu_closed: isNavHide}">
```

Comment:
Don't add logic into /html folder - create menu template in angular app folders!